datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/post/DeletePostResolver.java

```diff
@@ -0,0 +1,42 @@
+package com.linkedin.datahub.graphql.resolvers.post;
+
+import com.datahub.authentication.Authentication;
+import com.datahub.authentication.post.PostService;
+import com.linkedin.common.urn.Urn;
+import com.linkedin.common.urn.UrnUtils;
+import com.linkedin.datahub.graphql.QueryContext;
+import com.linkedin.datahub.graphql.authorization.AuthorizationUtils;
+import com.linkedin.datahub.graphql.exception.AuthorizationException;
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+import java.util.concurrent.CompletableFuture;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+
+@Slf4j
+@RequiredArgsConstructor
+public class DeletePostResolver implements DataFetcher<CompletableFuture<Boolean>> {
+  private final PostService _postService;
+
+  @Override
+  public CompletableFuture<Boolean> get(final DataFetchingEnvironment environment) throws Exception {
+    final QueryContext context = environment.getContext();
+
+    if (!AuthorizationUtils.canCreateGlobalAnnouncements(context)) {
```

Comment:
In the future, once the scope of Posts expands, we will need to change this permission. Maybe it's better to check the owner of the post.

Reply:
This addresses the use case for now, and we can get it in immediately. But I'd like to discuss this one offline a bit more.